dvc/repo/__init__.py

```diff
@@ -314,8 +314,8 @@ def __repr__(self):
     def find_root(cls, root=None, fs=None) -> str:
         from dvc.fs.local import LocalFileSystem, localfs
 
-        root_dir = os.path.realpath(root or os.curdir)
-        _fs = fs
+        root = root or os.curdir
+        root_dir = os.path.realpath(root)
```

Comment:
Need to get rid of this to detouch from fs. Some parts are not ready to switch away from abspaths yet, so keeping as is.